src/parallel_parameter_search/simulators.py

```diff
@@ -172,15 +178,15 @@ def __init__(self, namespace, gui, robot="wolfgang", ros_active=False, world="ro
             self.true_odom_publisher = rospy.Publisher(namespace + "/true_odom", Odometry, queue_size=1)
         path = rospack.get_path("wolfgang_webots_sim")
 
-        arguments = ["webots",
-                     "--batch",
-                     path + "/worlds/" + world + ".wbt"]
-        if not gui:
-            arguments.append("--minimize")
-            arguments.append("--no-rendering")
-        self.sim_proc = subprocess.Popen(arguments, stdout=subprocess.PIPE)
-
-        os.environ["WEBOTS_PID"] = str(self.sim_proc.pid)
+        if start_webots:
```

Comment:
Why do you want to initialize a webots sim, without starting the simulator?

Reply:
when you already started the simulator manually it can still connect to it. this is especially helpful during debugging since you don't need to restart the simulation each time again, thus leading to faster start up time